invoke.yml.example

```diff
@@ -2,14 +2,13 @@
 nautobot:
   project_name: "nautobot"
```

Comment:
Let's remove this while you're in here - no reason to override this normally

Reply:
I'll comment it out, but funny enough this is the exact item I wanted to change. I want be able to namespace it so I can create an ephemeral dev environment on the same box as my normal dev environment so I can destroy it at will without breaking my normal dev environment.